### PR TITLE
Nv 2795 allow to create integration for novu sms and email providers

### PR DESCRIPTION
**Pull request by BiswaViraj**
### What change does this PR introduce?

- allows the user to create Novu email and SMS integrations
- each provider is limited to 1 for each environment
### Why was this change needed?
- After deleting a Novu provider there was no option to re-add them.

### CODE REVIEW

File: apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export class CreateIntegration {
     if (command.providerId === SmsProviderIdEnum.Novu || command.providerId === EmailProviderIdEnum.Novu) {
       const count = await this.integrationRepository.count({
         _environmentId: command.environmentId,
-        providerId: EmailProviderIdEnum.Novu,
```
Comment (LetItRock) [inline apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts:117]: 🙌

File: apps/web/src/pages/integrations/components/multi-provider/CreateProviderInstanceSidebar.tsx [hunk 2]
```diff
@@ -67,8 +75,13 @@ export function CreateProviderInstanceSidebar({
...
   const selectedEnvironmentId = watch('environmentId');
 
-  const showInAppErrorMessage = useMemo(() => {
-    if (!provider || integrations.length === 0 || provider.id !== InAppProviderIdEnum.Novu) {
+  const showNovuProvidersErrorMessage = useMemo(() => {
+    const novuProviders: ProvidersIdEnum[] = [
+      InAppProviderIdEnum.Novu,
+      SmsProviderIdEnum.Novu,
+      EmailProviderIdEnum.Novu,
+    ];
```
Comment (LetItRock) [inline apps/web/src/pages/integrations/components/multi-provider/CreateProviderInstanceSidebar.tsx:83]: we have a const in shared package `NOVU_PROVIDERS`

**Review by LetItRock — APPROVED** [review]
awesome 🎉

File: apps/web/src/pages/integrations/components/multi-provider/CreateProviderInstanceSidebar.tsx (hunk 2 above)
Comment (BiswaViraj) [reply apps/web/src/pages/integrations/components/multi-provider/CreateProviderInstanceSidebar.tsx:83]: Ah nice, let me update it

**Pull request merged**